content/2022-02-10-brigade-2.3.0.md

```diff
@@ -0,0 +1,84 @@
+---
+title: "Brigade 2.3.0"
+slug: "2022-02-10-brigade-2.2.0"
+description: "Brigade 2.3.0 has been released"
+date: "2022-02-10"
+author: "Kent Rancourt"
+summary: "Brigade 2.3.0 has been released."
+---
+
+Good day, Brigadiers!
+
+Today, we're excited to announce our feature-packed v2.3.0
+release! Complete details are available in the
+[release notes](https://github.com/brigadecore/brigade/releases/tag/v2.3.0),
+of course, but since the
+[latest episode](https://www.youtube.com/watch?v=jCKf7YKiGjg) of our
+"Brigade in Five Minutes" series focused on authentication using third-party
+identity providers, and because we anticipate the next episode covering
+management of user permissions, now seems like an appropriate time to turn the
+spotlight on a specific new feature that streamlines that process.
+
+Beginning with Brigade v2.3.0, if you leverage a third-party identity provider
+like GitHub or an Azure Active Directory tenant to authenticate your users,
+there is now an option to _automatically_ grant users global, read-only
+permissions the first time they federate. This can be a tremendous convenience
+for Brigade administrators who already have a good handle on _who_ can
```

Comment:
```suggestion
for Brigade administrators that already have a good handle on _who_ can
```

Reply:
Not sure if this is the best suggestion but it feels weird to have two "who"s so close to each other.

Reply:
Fixed in #e77921ec5ed516e962496474ae6f842126506a72